test/evolve.test.mjs

```
@@ -7,10 +7,43 @@ import {
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export const AOS_ANT_OLD_WASM = fs.readFileSync(
+  path.join(
+    __dirname,
+    '../tools/fixtures/aos-ant-gFNGk70U9iwof9MyDqqKeizGa98w66-qCyFxupzXUnU.wasm',
+  ), // MUST load as binary (note no utf-8 flag)
+);
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)